docs/source/migrations/modes.rst

```diff
@@ -0,0 +1,55 @@
+======================
+Snuba Migration Modes
+======================
+
+This doc outlines a basic way to try out distributed and local migrations.
+Note that this is experimental, and should be used only for development
+purposes at the moment.
```

Comment:
Can we rephrase this to say that only local (single node) migrations are currently supported. And it's only the distributed migrations that should only be used for development purposes. This line makes it sound like both modes are experimental.

Reply:
Clarified it, thank you for pointing this out.